packages/manager/src/features/CloudPulse/Alerts/AlertsResources/AlertsResourcesNotice.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Notice } from '@linode/ui';
+import { styled, useTheme } from '@mui/material/styles';
+import React from 'react';
+
+interface AlertResourceNoticeProps {
+  handleSelectionChange: () => void;
+  selectedResources: number;
+  totalResources: number;
+}
+
+export const AlertsResourcesNotice = React.memo(
+  (props: AlertResourceNoticeProps) => {
+    const { handleSelectionChange, selectedResources, totalResources } = props;
+
+    const theme = useTheme();
+    return (
+      <StyledNotice
+        sx={{
+          height: theme.spacing(6.75),
+        }}
+        variant="info"
+      >
+        <b>
+          {selectedResources} of {totalResources} resources are selected.{' '}
+        </b>
+        {selectedResources !== totalResources && (
+          <StyledButton
+            aria-label="Select All Resources"
+            onClick={handleSelectionChange}
+            variant="text"
+          >
+            Select All
+          </StyledButton>
+        )}
+        {selectedResources === totalResources && (
+          <StyledButton
+            aria-label="Unselect All Resources"
+            onClick={handleSelectionChange}
+            variant="text"
+          >
+            Unselect All
+          </StyledButton>
+        )}
+      </StyledNotice>
+    );
+  }
+);
+
+const StyledNotice = styled(Notice, { label: 'StyledNotice' })(({ theme }) => ({
+  '&&': {
+    p: {
+      lineHeight: '1.25rem',
+    },
+  },
+  alignItems: 'center',
+  background: theme.bg.bgPaper,
+  borderRadius: 1,
+  display: 'flex',
+  flexFlow: 'row nowrap',
```

Comment:
```suggestion
  flexWrap: 'nowrap'
```

flex always by default row only